src/app/static/src/app/components/Stepper.vue

```diff
@@ -69,26 +69,21 @@
                 activeStep: state => state.activeStep,
                 steps: state => state.steps
             }),
-            ready: function () {
-                return this.$store.getters['stepper/ready']
-            },
-            complete: function (): CompleteStatus {
-                return this.$store.getters['stepper/complete']
-            }
+            ...mapGetters(namespace, ["ready", "complete"])
         },
         methods: {
-            ...mapActions(namespace, {
-                jump: 'jump',
-                next: 'next',
-                load: 'load'
-            }),
-            active(num: number) {
-                return this.activeStep == num;
+            ...mapActions(namespace, ["jump", "next"]),
+            ...mapActions(["reload"]),
+            isActive(num: number) {
+                return this.ready && this.activeStep == num;
             },
-            enabled(num: number) {
-                return this.steps.slice(0, num)
+            isEnabled(num: number) {
+                return this.ready && this.steps.slice(0, num)
```

Comment:
Changed the active/enabled/complete states so that they only show up once the data is loaded, to avoid odd user experience of some steps being coloured  before others.